src/app/components/Nav/Nav.scss

```diff
@@ -1,3 +1,5 @@
+@import 'app/styles/variables';
```

Comment:
One last comment: Running it on my computer, I had issues with these sass imports. I think you might have vscode setting that allows imports from an absolute path but this didn't work for me.

I had to change this to 
```@import '../../styles/variables';```

If you could update the setting into this repo to allow absolute paths, (i think it would be a jsconfig file but i have no idea). Either way, if the deploy script works then it's good for me but we might run into issues down the line.

Reply:
ahh sorry, I was in the middle of cleaning up all of the imports and must not have fully isolated my changes there. I'll be submitting another PR shortly which simplifies all imports